src/reference/crash-reporting.md

```diff
@@ -25,6 +25,9 @@ over time. A crash report may contain the following information:
 Google handles all data reported by this tool in accordance with the 
 [Google Privacy Policy][].
 
+Additionally, you may review the recently reported data in the 
+`dart-flutter-telemetry.log` file on your disk.
+
 ## Disabling analytics reporting
```

Comment:
I would mention it is also located in the user's HOME directory under the `.dart-tool` directory.. for example on macOS, you can find it at `$HOME/.dart-tool/dart-flutter-telemetry.log`

Reply:
Added!